store/tikv/retry/backoff.go

```diff
@@ -309,30 +135,30 @@ func (b *Backoffer) BackoffWithMaxSleep(typ BackoffType, maxSleepMs int, err err
 		}
 		logutil.BgLogger().Warn(errMsg)
 		// Use the first backoff type to generate a MySQL error.
-		return b.types[0].(BackoffType).TError()
+		return b.types[0].(*Config).err
```

Comment:
can we changes `types` to `configs []*Config`? It seems `*Config` is the only type.